pkg/v1/providers/ytt/02_addons/secretgen-controller/add_secretgen-controller.yaml

```diff
@@ -0,0 +1,24 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:template", "template")
+#@ load("@ytt:data", "data")
+#@ load("@ytt:yaml", "yaml")
+#@ load("/lib/helpers.star", "ValuesFormatStr")
+
+#@ if data.values.PROVIDER_TYPE != "tkg-service-vsphere" and data.values.SECRETGEN_CONTROLLER_ENABLE:
```

Comment:
question: Do we need to add secret-gen controller to both management cluster and workload cluster? 
If so, no issue here.

Reply:
Yes both mgmt cluster and workload cluster